packages/markers/src/browser/marker-tree-model.ts

```diff
@@ -23,6 +23,12 @@ export class MarkerTreeModel extends TreeModelImpl {
 
     @inject(OpenerService) protected readonly openerService: OpenerService;
 
+    selectNode(node: TreeNode): void {
```

Comment:
I don't think it is a proper place, it will break state in the tree selection service.